src/lib/forward.ts

```diff
@@ -138,9 +138,17 @@ export class EventForwarder {
           // include metadata and ccVersion in the response
           const metadata = node.getValueMetadata(args);
           args.metadata = metadata;
-          args.ccVersion =
-            node.getEndpoint(args.endpoint).getCCVersion(args.commandClass) ||
-            node.getEndpoint(0).getCCVersion(args.commandClass);
+          try {
+            args.ccVersion =
+              node.getEndpoint(args.endpoint).getCCVersion(args.commandClass) ||
+              node.getEndpoint(0).getCCVersion(args.commandClass);
```

Comment:
```suggestion
              node.getEndpoint(args.endpoint)?.getCCVersion(args.commandClass) ||
              node.getEndpoint(0)?.getCCVersion(args.commandClass);
```

In this way you can remove the try/catch

Reply:
isn't endpoint 0 always available as it's the node itself ?

Reply:
> isn't endpoint 0 always available as it's the node itself ?

Maybe yes, don't remember, but it doesn't add any overhead so I would keep the `?` :)